release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -37,8 +37,62 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### `lang.xml:data()` Lang Library Function
+
+The `lang.xml:data()`  lang library function has been introduced aiming to replace the usage of the `lang.xml:getContent()` function to get the character data from an `xml` value. This function returns the character data of a `xml:Text` value as a string. For an `xml:Element` value this returns the concatenation of the result of invoking this function over the children of the element. Calling `lang.xml:data()` on comments, processing instructions, and empty sequences returns an empty string.
+
+##### `lang.xml:getDescendants()` Lang Library Function
+
+The `lang.xml:getDescendants()` lang library function has been introduced to retrieve a sequence of the descendant elements for a given element, in document order.
```

Comment:
```suggestion
The `lang.xml:getDescendants()` lang library function has been introduced to retrieve a sequence of the descendant elements for a given element in the documented order.
```

Reply:
I believe "document order" is used here as a specific XML term. See "document order" in https://www.w3.org/TR/2004/REC-DOM-Level-3-Core-20040407/glossary.html.

@pcnfernando @rdhananjaya can you please confirm?

Reply:
Yes, it's taken as in the XML context